src/steps/findings/converter.ts

```diff
@@ -26,19 +26,18 @@ const staticScanExploitabilityMap = {
   '2': 'Very Likely',
 };
 
-export function createFindingEntity(
-  veracodeFinding: Finding,
-  applicationName: String,
-): Entity {
+export function createFindingEntity(veracodeFinding: Finding): Entity {
+  const name = `cwe-${veracodeFinding.finding_details.cwe.id}: ${veracodeFinding.finding_details.file_name} (Line: ${veracodeFinding.finding_details.file_line_number})`;
```

Comment:
Are these objects (finding_details & cwe) guaranteed to be present?

Reply:
yup! for static scans at least